data/human/free worlds start.txt

```diff
@@ -616,10 +616,18 @@ mission "FW Hope Recon 1C"
 			`	The landing crew has absolutely nowhere to hide in this glacial field; you pick them off easily. There is no response from the ship, so you land beside it and prepare to board it. But, the moment you open your hatch, the other ship roars to life, powering up its engines and beating a hasty retreat while firing a few parting shots at you from its turret. You fire up your own engines and pursue it...`
 				launch
 			
-			label spare
+			label accept
 			apply
 				karma ++
-			`	Most ships are equipped with external speakers for exactly this sort of situation. Speaking through them, you tell the Navy crew below, "Return to your ship and leave this system immediately." They beat a hasty retreat and fire up their engines. You follow their ship into orbit to make sure it truly does leave the system...`
+				set "fw accepted Susquehanna's word of honor"
+			`	Most ships are equipped with external speakers for exactly this sort of situation.`
+			`	"Crew of the R.N.S. Susquehanna, this is Captain <last> of the Free Worlds ship <ship>," you announce. "Will you each give me your word of honor that you will take no further part in fighting the Free Worlds?"`
+			`	The Gunboat captain thinks for a brief moment and gives you a thumbs-up sign, their hand held high in the air. Their crew follows suit without any visible hesitation.`
+			`	You consider the situation and decide not to bother with landing and taking your new prisoners on board.`
```

Comment:
This line seems unnecessary to me. If they've given their word not to fight, why would you need to think about landing to take prisoners?

Reply:
> This line seems unnecessary to me.

Hmm, yes. It's backward, isn't it? This sentence should come before offering the parole deal to the Navy soldiers, not after.

> If they've given their word not to fight, why would you need to think about landing to take prisoners?

Because the player might not be willing to trust a random Navy captain's word enough to let them take off in a fully armed Gunboat, at this point? In the Kornephoros prisoners exchange, the Navy soldiers were returned to Republic space in FW ships, and the cease fire enabling this was negotiated by Alondo with... it doesn't say with whom, but I guess it would have been the Navy brass or the Republic government, not just a random captain.